### PR TITLE
Identity Beta release June 2020 changelog updates

### PR DESCRIPTION
**Pull request by g2vinay**

### CODE REVIEW

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 ## 1.1.0-beta.5 (Unreleased)
 
+### New Features
+- Added option to write to shared token cache from `ClientSecretCredential`, `ClientCertificateCredential.
```
Comment (alzimmermsft) [inline sdk/identity/azure-identity/CHANGELOG.md:6]:
```suggestion
- Added option to write to shared token cache from `ClientSecretCredential`, `ClientCertificateCredential`.
```

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+- Added new Developer credentials `IntelliJCredential`, `VsCodeCredential` and `AzureCliCredential`.
```
Comment (alzimmermsft) [inline sdk/identity/azure-identity/CHANGELOG.md:7]:
```suggestion
- Added new developer credentials `IntelliJCredential`, `VsCodeCredential` and `AzureCliCredential`.
```

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,19 @@
...
+- First preview of new API for authenticating users with `DeviceCodeCredential` and `InteractiveBrowserCredential`
```
Comment (alzimmermsft) [inline sdk/identity/azure-identity/CHANGELOG.md:8]: Probably don't need to call out that it is the first preview, can simply state

```suggestion
- New APIs for authenticating users with `DeviceCodeCredential` and `InteractiveBrowserCredential`.
```

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,19 @@
...
+- Added method `authenticate` which pro-actively interacts with the user to authenticate if necessary and returns a serializable `AuthenticationRecord`
```
Comment (alzimmermsft) [inline sdk/identity/azure-identity/CHANGELOG.md:9]: Was this new method added to a top level interface or to specific classes?

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,19 @@
...
+- Added following configurable options in classes `DeviceCodeCredentialBuilder` and `InteractiveBrowserCredentialBuilder`
+    - `authenticationRecord` enables initializing a credential with an `AuthenticationRecord` returned from a prior call to `Authenticate`
+    - `disableAutomaticAuthentication` disables automatic user interaction causing the credential to throw an `AuthenticationRequiredException` when interactive authentication is necessary.
+
+### Breaking Changes
+- Removed support to disable specific credentials in `DefaultAzureCredential` authentication flow.
```
Comment (jianghaolu) [inline sdk/identity/azure-identity/CHANGELOG.md:15]: Removed support to exclude specific credentials?

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 4 above)
Comment (g2vinay) [reply sdk/identity/azure-identity/CHANGELOG.md]: Added a tab to this text, it is supposed to be under the previous comment.

**Pull request merged**